### PR TITLE
YLP-878 Wrong settings for glucose units uploaded by the handset

### PR DESCRIPTION
**Pull request by yvon-dblg**

### CODE REVIEW

File: packages/yourloops/lib/utils.ts [hunk 1]
```diff
@@ -123,3 +124,31 @@ export function getUserInitials(user: IUser): string {
...
+ * @returns Fixed settings
+ */
+export function fixYLP878Settings(settings: Settings | undefined | null): Settings {
+  if (_.isNil(settings)) {
+    return {
+      country: "FR",
+      units: {
+        bg: Units.gram,
+      },
+    };
```
Comment (franck-fourel) [inline packages/yourloops/lib/utils.ts:140]: why don't we have a default object that we merge with incoming values: wondering if the lodash.merge could make the job to produce the newSettings ?

**Review by franck-fourel — COMMENTED** [review]
it works as expected
a quick comment on the way the fixYLP878Settings is written. 
I can accept it as it is if you think it won't be more readable with objects merge.

File: packages/yourloops/lib/utils.ts (hunk 1 above)
Comment (yvon-dblg) [reply packages/yourloops/lib/utils.ts:140]: It was to cleanup the settings we used.
To be sure we only have what we want.

**Review by franck-fourel — CHANGES_REQUESTED** [review]
LGTM

**Review by franck-fourel — APPROVED** [review]

**Pull request merged**